configs/chrome/README.md

```diff
@@ -0,0 +1,7 @@
+# chrome configuration readme 
+  * enable_create_new_profile : `true/false, will create new chrome profile before running the whole suite ` 
```

Comment:
* enable_create_new_profile : `true/false, it will create new chrome profile before running the whole suite if enabled `